test/napi/napi-app/test_wrap_cleanup_order.c

```
@@ -0,0 +1,86 @@
+    int child_count = 32;
+    if (argc >= 1) {
+        napi_get_value_int32(env, args[0], &child_count);
+    }
+
+    napi_value result;
+    napi_create_array(env, &result);
+
+    napi_value parent;
+    napi_create_object(env, &parent);
+    int* parent_id = (int*)malloc(sizeof(int));
+    *parent_id = 0;
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)